### PR TITLE
Fix pageFooter option and use double-bang on customMetadata

### LINKED ISSUE #184: The pageFooter option is broken
Caused by #180, see in particular commit https://github.com/wswebcreation/multiple-cucumber-html-reporter/pull/180/commits/3cda95b2b20b5009ebd147cd77b7d7d04f22596b.

While the other parameters are `boolean` the `pageFooter` is indeed a `string`. Hence, applying the double bang operator causes a forced conversation to `boolean` which is _not_ what we want. This causes the `pageFooter` to be converted to `true` when a custom `HTML` code is passed. Causing, in turn, to display a `true` footer text.

**Environment (please complete the following information):**
 - multiple-cucumber-html-reporter: 1.21.0
 - Node.js version: 14.19.0
 - Yarl version: 1.22.18
 - Platform name and version: Linux (amx64/armhf)
 - Cucumber version: 6.0.5

**Expected behavior**
The string is kept as is so that we can pass a custom footer.

### PR DESCRIPTION
**Pull request by BaCaRoZzo**
Fixes #184 

Moves the `pageFooter` option back while aligning `customMetadata` to the other `boolean` options. I've also hidden the darkmode toggle in the custom `CSS` example since it doesn't make sense there.

### CODE REVIEW

File: test/custom.css [hunk 1]
```diff
@@ -12,6 +12,11 @@ body {
   --skipped-color:#770077;
 }
 
+/* Hide darkmode toggle button */
+.fa-toggle-off {
```
Comment (wswebcreation) [inline test/custom.css:16]: Hi @BaCaRoZzo 

Not sure why you want to hide the button, can you explain the use case? Hiding this one would prevent toggling back

**Review by wswebcreation — COMMENTED** [review]
Thanks for the fix, I have one small comment. Based on your answer I can merge and release it

File: test/custom.css (hunk 1 above)
Comment (BaCaRoZzo) [reply test/custom.css:16]: @wswebcreation when a custom theming is applied the toggle may or may not work, depending on the theming. In my case it didn't work so I hidden for myself.

I thought providing the user with a quick way to hide it was valuable as well and thus I've added it. Ofc I can remove it, no biggie. The important bit is the fix, indeed. :slightly_smiling_face:

Comment (Baskercarrer) [reply test/custom.css:16]: I agree, hiding toggle-off in custom.css is good one since it doesn't required to show it as it is custom anyway.

Comment (wswebcreation) [reply test/custom.css:16]: Thanks for explaining!

**Review by wswebcreation — APPROVED** [review]

**Pull request merged**